flink-table/flink-table-planner/src/test/scala/org/apache/flink/table/planner/plan/rules/logical/MultiJoinOptimizeTest.scala

```diff
@@ -0,0 +1,170 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.apache.flink.table.planner.plan.rules.logical
+
+import org.apache.flink.table.catalog.ObjectPath
+import org.apache.flink.table.catalog.stats.CatalogTableStatistics
+import org.apache.flink.table.planner.plan.optimize.program._
+import org.apache.flink.table.planner.utils.{TableConfigUtils, TableTestBase}
+
+import org.apache.calcite.plan.hep.HepMatchOrder
+import org.apache.calcite.rel.rules.CoreRules
+import org.apache.calcite.tools.RuleSets
+import org.junit.{Before, Test}
+
+/**
+ * Plan tests for join order using rule [[org.apache.calcite.rel.rules.LoptOptimizeJoinRule]] after
+ * joins converts to multi join set by
+ * [[org.apache.flink.table.planner.plan.rules.logical.FlinkJoinToMultiJoinRule]].
+ */
+class MultiJoinOptimizeTest extends TableTestBase {
```

Comment:
If we want to test join reorder, please add test cases into JoinReorderTestBase

Reply:
> If we want to test join reorder, please add test cases into JoinReorderTestBase

done!